go/vt/vttablet/tabletmanager/vreplication/vplayer.go

```diff
@@ -563,6 +563,8 @@ func (vp *vplayer) applyEvent(ctx context.Context, event *binlogdatapb.VEvent, m
 		return io.EOF
 	case binlogdatapb.VEventType_HEARTBEAT:
 		// No-op: heartbeat timings are calculated in outer loop.
+	case binlogdatapb.VEventType_SAVEPOINT:
```

Comment:
I believe we have to apply all savepoint statements. Otherwise, the target will diverge, right?
In the tests, I see that there is no rollback to savepoint, where this becomes more relevant.

Reply:
I see that you've renamed the fields. All that's left is apply the statements here?

Reply:
I believe these changes should apply savepoints: vplayer.go: 192, vstreamer.go 247. Tests added for both vrepl/vstream client.